iroh-cli/src/commands.rs

```diff
@@ -22,8 +23,9 @@ pub(crate) mod rpc;
 pub(crate) mod start;
 pub(crate) mod tags;
 
-/// iroh is a tool for building distributed apps
-/// https://iroh.computer/docs
+/// iroh is a tool for building distributed apps.
+///
+/// For more information, visit: https://iroh.computer/docs.
```

Comment:
```suggestion
/// For more information, visit: <https://iroh.computer/docs>.
```

I'm not 100% sure, but that trailing dot *may* be interpreted as part of the URL. And if so, the URL may 404. This is safer :)